styles.css

```diff
@@ -37,20 +43,57 @@ body {
     }
 
     .icon-menu {
-      #img-1 {
-        width: 18px;
-        height: 12px;
-        display: flex;
-        align-items: center;
-        flex: none;
-        order: 0;
-        flex-grow: 0;
-        text-decoration: none;
+      #icon-1 {
+        color: #fff;
+        font-size: x-large;
+        cursor: pointer;
       }
 
       .img-1-nav {
-        display: none;
+        background-color: #3c3a39;
         list-style: none;
+        position: absolute;
+        left: 0;
+        top: 0;
+        width: 100%;
+        height: 745px;
+        z-index: 99;
+        margin: 0;
+        padding: 0;
+        visibility: hidden;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        align-items: flex-start;
```

Comment:
- [ ] Great job 👏 implementing the hamburger menu as specified in the project requirement. There is still a little issue 🐞 with the UI which we need to fix. Currently, when a user opens the hamburger menu and scrolls on the page there get to see the main page which shouldn't be so. Please note that when the menu is opened the user is not supposed to see anything on the main page apart from the menu. See below for more details:
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="374" alt="Screenshot 2023-06-20 at 09 41 29" src="https://github.com/2004-okari/MY-PORTFOLIO/assets/107192076/742efd6d-f1ff-4882-b90a-2806699f6f91">
</td>
<td><img width="374" alt="Screenshot 2023-06-20 at 09 28 32" src="https://github.com/2004-okari/MY-PORTFOLIO/assets/107192076/2eeab25e-01e1-486f-8b47-b1b43b8fd4f5"></td>
</tr>
</table>